### ISSUE TITLE
Errors when doing make on debian 7

### ISSUE THREAD

**Issue by mumbaumba** [reorg/pg_repack#75]
We are migrating from postgres 9.1 to 9.3 & I've decided to install pg_repack
So I've downloaded, unpacked re_pack v1.3.3
Set the  pg_config of PostgreSQL 9.3 in the PATH, so that typing pg_config gives me:
pg_config is /usr/lib/postgresql/9.3/bin/pg_config
But when I'm doing make, I receive:
 Entering directory `/root/pg_repack-1.3.3/lib'
gcc -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -I/usr/include/mit-krb5 -fPIC -pie -DLINUX_OOM_SCORE_ADJ=0 -fno-omit-frame-pointer -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -g -fpic -shared -Wl,--version-script=exports.list -o pg_repack.so repack.o pgut/pgut-be.o pgut/pgut-spi.o -L/usr/lib/x86_64-linux-gnu -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -L/usr/lib/mit-krb5 -L/usr/lib/x86_64-linux-gnu/mit-krb5 -L/usr/lib  -Wl,--as-needed  
/usr/bin/ld:exports.list:2: syntax error in VERSION script
collect2: error: ld returned 1 exit status

 When I'm trying :
make PG_CONGIG=/usr/lib/postgresql/9.3/bin/pg_config --debug=v
I get:
/usr/include/postgresql/internal/pqexpbuffer.h:149:13: error: old-style parameter declarations in prototyped function definition
pg_repack.c:2002:1: error: expected '{' at end of input
make[1]: **\* [pg_repack.o] Error 1
make[1]: Leaving directory `/root/pg_repack-1.3.1/bin'
make: **\* [all] Error 2
I would be very thankful for help.

**Comment by mumbaumba** [comment]
More digging with that issue.
I've made 2 new clean installations on 2 separate comps:
1) debian 7 with postgres 9.1 
    repack installed easily
2) debian 7 with postgres 9.3 from apt.postgresql.org
    and again it : 
  make --debug=v (with v  for  more  verbose  basic
            debugging) gives me the same mistake:
 /usr/include/postgresql/internal/pqexpbuffer.h:149:13: error: old-style parameter declarations in prototyped function definition
pg_repack.c:2002:1: error: expected '{' at end of input
make[1]: *** [pg_repack.o] Error 1
make[1]: Leaving directory `/root/pg_repack-1.3.1/bin'
make: *** [all] Error 2
More information from debug can be found in the attached file
[errors.txt](https://github.com/reorg/pg_repack/files/157675/errors.txt)

**Comment by schmiddy** [comment]
Hm, I think there have been some issues reported (see #68) with apt.postgresql.org Debian packages causing mismatched versions of libpq and postgresql to be installed. Can you check if something like that has happened, try `dpkg -l | grep -E '(postgres|libpq)'`

**Comment by demofly** [comment]
I've met exactly the same bug.

Even `dpkg -l | grep -E '(postgres|libpq)'` didn't help. I've switched to make a backport from https://packages.debian.org/source/stretch/pg-repack
But, met the same bug. But while debuild worked, I've noticed this error:
```
( echo '{ global:'; mawk '/^[^#]/ {printf "%s;\n",$1}' exports.txt; echo ' local: *; };' ) >exports.list
/bin/sh: 1: mawk: not found
```

So, `apt-get -y install mawk postgresql-server-dev-all` helped as well and resolved the issue.
